tests/west/project/test_project.py

```diff
@@ -38,8 +40,7 @@ def repos_tmpdir(tmpdir):
     ├── west (branch: master)
     │   └── (contains this west's worktree contents)
     ├── manifest (branch: master)
-    │   └── manifest.yml
-    ├── Kconfiglib (branch: zephyr)
```

Comment:
It doesn't look like Kconfiglib project should be deleted here. It's still a project which gets created and put into the manifest, and there is still a separate manifest repository. Were these lines changed in error?

Reply:
Fixed.